docker-hub/builds/index.md

```diff
@@ -23,19 +23,24 @@ to a source code branch (for example in GitHub) for one of those listed image
 tags, the push uses a webhook to trigger a new build, which produces a Docker
 image. The built image is then pushed to the Docker Hub registry.
 
-> **Note**: You can still use `docker push` to push pre-built images to
+> **Note**
+>
+> You can still use `docker push` to push pre-built images to
 repositories with Automated Builds configured.
 
 If you have automated tests configured, these run after building but before
 pushing to the registry. You can use these tests to create a continuous
 integration workflow where a build that fails its tests does not push the built
 image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
 
-Depending on your plan, you may get parallel builds, which means that `N`
+Depending on your [plan](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}, you may get parallel builds, which means that `N`
 autobuilds can be run at the same time. `N` is configured by the plan that you
 subscribe to. Once `N+1` builds are running, any additional builds are
 queued to be run later.
```

Comment:
looks like the added line is missing :)

Reply:
@arkodg I've added it as a Note below this :)

Reply:
perfect thanks !